docs/using-latest-daily.md

```diff
@@ -6,33 +6,24 @@ These instructions will get you set up with the latest build of Aspire. If you j
 
 See [machine-requirements.md](machine-requirements.md).
 
-## Run the workload installation script
-
-The [workload installation script](./../eng/installLatestFromReleaseBranch.ps1)) will install the latest .NET Aspire workload from the release branch, but it can also install latest from main (latest nightly build) if the `-FromMain` flag is used (`--fromMain` on Linux/macOS).
-
-### Windows
-
-From a powershell prompt, and from the root of the aspire repo, run:
-
-```shell
-.\eng\installLatestFromReleaseBranch.ps1 -FromMain
-```
-
-### Linux/macOS
-
-From a terminal, and from the root of the aspire repo, run:
-
-```shell
-./eng/installLatestFromReleaseBranch.sh --fromMain
-```
-
 ## Add necessary NuGet feeds
 
 The latest builds are pushed to a special feed, which you need to add:
 ```sh
 dotnet nuget add source --name dotnet8 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json
 ```
 
+If you use package source mapping, you'll also need to add the following mappings to your NuGet.config
+
+```xml
+<packageSourceMapping>
+  <packageSource key="dotnet8">
+    <package pattern="Aspire.*" />
+    <package pattern="Microsoft.Extensions.*" />
+  </packageSource>
+</packageSourceMapping>
```

Comment:
Could you consider adding the entire NuGet.config example?

I did not have one and had to look at the .NET 9 repo for how it was configured, I finished with something like this at the end.

```suggestion
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
    <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
  </packageSources>
  <packageSourceMapping>
    <packageSource key="dotnet8">
      <package pattern="Aspire.*" />
      <package pattern="Microsoft.Extensions.*" />
    </packageSource>
  </packageSourceMapping>
</configuration>

```

Reply:
The `dotnet nuget add source` should have added the appropriate Package Source to your nuget.config file, although perhaps it did it in your user profile's file (E.g. `%appdata%\NuGet\nuget.config`)

The Package Source Mappings are only needed if you use package source mappings - if you didn't already have a nuget.config file, then you probably don't need the mappings.  If you do have mappings, then you'll need to carefully merge the mappings into your existing mappings.  Because of this, I'm hesitant to provide a complete `nuget.config`.  If I used your suggested nuget.config, I would be in violation of my company's internal policies which prohibit the use of nuget.org, instead requiring us to go through an internal mirror which is configured in a higher level nuget.config.

I did add a link to Package Source Mappings however to make it easier to find out what they are and how to configure them.

Reply:
Understood, it was probably an issue on my side

Reply:
I think we should recommend creating a local nuget.config file as updating your global feeds to include these CI feeds is generally not a great idea.